posts/espp-in-this-economy/index.qmd

```diff
@@ -0,0 +1,215 @@
+---
+title: "ESPP In This Economy?"
+description: "Employee Stock Purchase Plans can generate thousands in extra income no matter what happens in the stock market."
+date: "2025-04-21"
+draft: true
+echo: false
+image: "image.png"
+format:
+  html: {}
+---
+
+```{dot}
+//| fig-cap: An ESPP allows employees to purchase shares at a discount with after-tax money.
+//| fig-width: 2
+//| fig-height: 5
+//| column: margin
+
+digraph G {
+    node [shape=plaintext]
+    splitrect [
+        label=<
+            <TABLE BORDER="1" CELLBORDER="1" CELLSPACING="0">
+                <TR>
+                    <TD ROWSPAN="2" BGCOLOR="lightgreen" WIDTH="35" HEIGHT="80" FIXEDSIZE="TRUE">Profit</TD>
+                    <TD BGCOLOR="lightblue" WIDTH="85" HEIGHT="30" FIXEDSIZE="TRUE">Discount</TD>
+                </TR>
+                <TR>
+                    <TD BGCOLOR="lightblue" WIDTH="85" HEIGHT="50" FIXEDSIZE="TRUE">Lookback<br/>(optional)</TD>
+                </TR>
+                <TR>
+                    <TD BGCOLOR="grey" WIDTH="35" HEIGHT="170" FIXEDSIZE="TRUE"></TD>
+                    <TD BGCOLOR="grey" WIDTH="85" HEIGHT="170" FIXEDSIZE="TRUE">After-Tax<br/>Contributions</TD>
+                </TR>
+            </TABLE>
+        >
+    ]
+}
+```
+
+Most people misunderstand Employee Stock Purchase Plans (ESPPs).
+
+Even during the COVID stock market crash, I contributed the maximum to my employer's ESPP.
```

Comment:
I could see someone reading this thinking, "Well, I can't afford not to have the income taken home immediately." I know the target audience for this blog likely _can_, but perhaps a couple of sentences reminding them that it's deferred income, or even a thought on how to weigh when to start contributing to ESPP (as in, no credit card debt, a healthy emergency fund, etc). I understand if you feel like that's beyond the scope here, but I wanted to raise the question.

Reply:
Thank you. I think is a good idea to mention the lock up part sooner.